lib/twine/formatters/android.rb

```diff
@@ -41,7 +41,7 @@ def determine_language_given_path(path)
       end
 
       def output_path_for_language(lang)
-        "values-#{lang}"
+        "values-#{lang}".gsub(/-(\p{Lu})/, '-r\1')
```

Comment:
Do we need to be this fancy? Wouldn't a `[A-Z]` do the job? It might be only me, but I had to look up `\p{Lu}`. Or, to the other extreme, aren't we matching a too large character group with this one? Aren't region codes ASCII only?